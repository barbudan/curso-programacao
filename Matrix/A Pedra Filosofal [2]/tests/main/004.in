10 10
0 0 0 0 0 0 0 0 0 0
0 0 5 0 0 0 0 0 0 0
0 4 8 0 0 0 0 0 0 0
0 0 10 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 8 9 0 0 0
0 0 10 10 0 9 0 0 0 0
0 0 5 8 0 0 0 0 8 10
0 0 0 0 0 0 0 10 0 10