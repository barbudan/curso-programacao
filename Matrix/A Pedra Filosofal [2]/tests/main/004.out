30 9 9
